draft-ietf-quic-transport.md

```diff
@@ -803,12 +803,14 @@ frame, and the receiver MUST use the final offset to account for all bytes sent
 on the stream in its connection level flow controller.
 
 RST_STREAM terminates one direction of a stream abruptly.  Whether any action or
-response can or should be taken on the data already received is an
-application-specific issue, but it will often be the case that upon receipt of a
-RST_STREAM an endpoint will choose to stop sending data in its own direction. If
-the sender of a RST_STREAM wishes to explicitly state that no future data will
-be processed, that endpoint MAY send a STOP_SENDING frame at the same time.
-
+response can or should be taken on the data already received is an application-
+specific issue.
+
+For a bidirectional stream, RST_STREAM has no effect on data flow in the
+opposite direction. Both endpoints MUST maintain state for that direction until
+that direction enters a terminal state, or either side sends CONNECTION_CLOSE or
+APP_CLOSE. The RST_STREAM sender MAY also send a STOP_SENDING frame to encourage
```

Comment:
I think this last sentence might be better as the second sentence.  I was a bit confused about whether you meant connection on stream termination when I first read it.